lib/rules/declaration-property-value-no-unknown/index.js

```diff
@@ -63,9 +65,47 @@ const rule = (primary, secondaryOptions) => {
 			return valuePattern && matchesStringOrRegExp(value, valuePattern);
 		};
 
-		const propertiesSyntax = (secondaryOptions && secondaryOptions.propertiesSyntax) || {};
+		const propertiesSyntax = {
+			// Take a shallow clone as this object will be appended to.
+			...((secondaryOptions && secondaryOptions.propertiesSyntax) || {}),
+		};
 		const typesSyntax = (secondaryOptions && secondaryOptions.typesSyntax) || {};
```

Comment:
[suggest] Using the `?.` operator can simplify the code a bit:

```suggestion
		const propertiesSyntax = {
			// Take a shallow clone as this object will be appended to.
			...(secondaryOptions?.propertiesSyntax ?? {}),
		};
		const typesSyntax = secondaryOptions?.typesSyntax ?? {};
```

Reply:
Considering consistency, it seems better to fix `ignorePrpoperties` too:

```diff
-Object.entries((secondaryOptions && secondaryOptions.ignoreProperties) || {})
+Object.entries(secondaryOptions?.ignoreProperties ?? {})
```

https://github.com/stylelint/stylelint/pull/7078/files#diff-e97a8a2ce49674252ada6c3a6f0dde2f19422f662d0afe69be865ddc97b418a5R57

But it's not required, and it's possible to keep it as-is. 👌🏼

Reply:
Thank you!